components/core/src/clp/clp/run.cpp

```
@@ -64,12 +64,22 @@ int run(int argc, char const* argv[]) {
+                auto optional_captures{lexer.get_captures_from_rule_id(rule_id)};
+                if (false == optional_captures.has_value()) {
+                    continue;
                 }
+
+                auto const& captures{optional_captures.value()};
+                if ("header" == rule_name && 1 == captures.size()
+                    && "timestamp" == captures[0]->get_name())
+                {
+                    continue;
+                }
+
```

Cppcheck (2.19.0)

[style] The function 'get_on_disk_size' is never used.

(unusedFunction)